### ISSUE TITLE
Question: how about the frontend folder/file naming convension when using inertia-rails?

### ISSUE THREAD

**Issue by xfyuan** [inertiajs/inertia-rails#122]
First, Thanks for your hard work to introduce the great gem to integrated the inertia.js into rails! 👍 

There're a few confused questions about the frontend folder/file naming convention by using it:

- What's the best way to naming the frontend folder? snakecase follow Rails way, or other?
- What's the best way to naming the frontend file? snakecase follow Rails way, or other?

And there're another question which is met from above points: 

**could we not to call the `render` in each controller method(such as `#show, #index`) expicitly, and inertia-rails can know which frontend file/component should be used through the folder/file/class/method namecast?** 

If we can, It seems to follow the Rails way perfectly.)

**Comment by arni1981** [comment]
The latter is mentioned in the docs https://github.com/inertiajs/inertia-rails?tab=readme-ov-file#rails-component-and-instance-props

```ruby
class EventsController < ApplicationController
  use_inertia_instance_props

  def index
    @events = Event.all
  end

end
```
...is the same as ...

```ruby
class EventsController < ApplicationController
  def index
    render inertia: 'events/index', props: {
      events: Event.all
    }
  end
end
```

**Comment by xfyuan** [comment]
> The latter is mentioned in the docs https://github.com/inertiajs/inertia-rails?tab=readme-ov-file#rails-component-and-instance-props
> 
> ```ruby
> class EventsController < ApplicationController
>   use_inertia_instance_props
> 
>   def index
>     @events = Event.all
>   end
> 
> end
> ```
> 
> ...is the same as ...
> 
> ```ruby
> class EventsController < ApplicationController
>   def index
>     render inertia: 'events/index', props: {
>       events: Event.all
>     }
>   end
> end
> ```

Ahh yes, that's it. Thanks for your so kindly reply.)

And how about the formmer question? It seems that there're different naming style in the README examples?(such as `Event/Index` and `events/index`)

**Comment by PedroAugustoRamalhoDuarte** [comment]
For now, we don't have a strong convention about that. In my opinion is better to use like `Event/Index` because is the name of the file in the frontend and they use , in React at least,  `PascalCase`.

[Inertia Rails contrib]( https://github.com/skryukov/inertia_rails-contrib) uses  `PascalCase` as well

**Comment by BrandonShar** [comment]
Solved via #144 !
